### ISSUE TITLE
Missing until params

### ISSUE THREAD

**Issue by mortharia** [docker-java/docker-java#1427]
According to https://docs.docker.com/engine/reference/commandline/container_logs/, we can use the "until" option in the docker logs command. Why this param is not supported in the [LogContainerCmd](https://github.com/docker-java/docker-java/blob/master/docker-java-api/src/main/java/com/github/dockerjava/api/command/LogContainerCmd.java)?

**Comment by bsideup** [comment]
You should be using at the API, not CLI:
https://docs.docker.com/engine/api/v1.40/#operation/ContainerLogs

the parameter is called `follow`.

**Comment by mortharia** [comment]
No, "follow" is used to keep connection after returning logs. I am talking about the "until" params!

**Comment by bsideup** [comment]
@mortharia sorry, I should have got my morning cup of coffee before reading the issues 😅 

yes, `until` seems to be missing. Do you want to contribute it?

**Comment by mortharia** [comment]
yes.

**Comment by mortharia** [comment]
It is done! would it be possible to include it in the next release, please?

**Comment by metcox** [comment]
Hi, 
is there any progress on this issue, I also need this feature. Thanks for your work, really helpful project.

**Comment by sawyersong2** [comment]
I also need this help, is there any progress? Thank you very much for your work.
